docs/20260130_1200_博報堂2.47兆円支出調査.md

```
@@ -0,0 +1,404 @@
+
+| 項目 | 内容 |
+|------|------|
+| 事業目的 | 電気・ガス料金の激変緩和対策等により、家計・企業等の負担を軽減 |
+| 点検結果 | 支援対象となる全ての需要家に支援が確実に届けられている。消費者物価指数にも物価抑制の効果が現れている |
+| 外部有識者点検 | 書面点検実施 |
+| 有識者からの指摘 | ・支出先上位リストの一者応札・随契理由の記載不備<br>・随意契約理由の記載方法の見直しが必要 |
+| 今後の方向性 | 事業終了予定。類似事業実施時には実施状況分析結果を踏まえて検討 |
+
+### 特設サイト
+
+- URL: https://denkigas-gekihenkanwa.go.jp/
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)